experiments/standalone/Soil/richards_runoff.jl

```diff
@@ -54,14 +71,103 @@ runoff_model = ClimaLand.Soil.Runoff.TOPMODELRunoff{FT}(;
     f_max = f_max,
     R_sb = R_sb,
 )
-vg_α = ClimaCore.Fields.ones(subsurface_space) .* FT(0.2)
-hydrology_cm = map(vg_α) do (α)
-    FT = typeof(α)
-    ClimaLand.Soil.vanGenuchten{FT}(; α = α, n = α + FT(2))
+soil_params_artifact_path = "/groups/esm/ClimaArtifacts/artifacts/soil_params_Gupta2020_2022"# @clima_artifact("soil_params_Gupta2020_2022")
+function mask_vg(var, value)
```

Comment:
this is stand in for now to prevent errors over the ocean, where the soil parameters are unphysical

Reply:
Maybe add this as comment?